package.json

```diff
@@ -254,10 +253,5 @@
   },
   "engines": {
     "node": "^8"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
```

Comment:
Not a big fan of this change @nirinchev.
Having a pre-commit hook makes it easier to help contributors remember to run linting.

If you think this was really necessary, I think it should have been a separate PR to make it easier to revert in the future.

Reply:
I couldn't commit due to an error that I showed you some time ago. While I agree it could have been done in a separate PR, it didn't seem worth the effort. I'll create an issue to track enabling it.

Reply:
☝️ did you create an issue to track enabling it @nirinchev? I can't seem to find it ..

Reply:
Of course - https://github.com/realm/realm-studio/issues/1166 - it's the latest issue in the repo 😝